pkg/apis/v1alpha1/kwokctl_configuration_types.go

```diff
@@ -259,7 +259,10 @@ type KwokctlConfigurationOptions struct {
 	// KubeSchedulerPort is kube-scheduler port in the binary runtime
 	KubeSchedulerPort uint32 `json:"kubeSchedulerPort,omitempty"`
 
-	// KwokControllerPort is kube-controller port in the binary runtime
+	// KwokControllerPort is kube-controller port
+	// is the default value for --controller-port or env KWOK_CONTROLLER_PORT
+	// if not set, in the binary runtime is choosen by net.GetUnusedPort,
```

Comment:
Not sure if every runtime should default to `10247` so I kept the current behaviour. Please let me know if I should change it so default is 10247 on all runtimes